app/routes.py

```
@@ -0,0 +1,40 @@
+        
+        return ComparisonResponse(
+            status="success",
+            message="Comparison completed successfully",
+            diff_file=f"updates/{result.repo_name}-{result.timestamp.strftime('%Y-%m-%d')}-open-api-diff.md"
+        )
+        
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+@router.get("/health")
+async def health_check():
```

Ruff (0.8.2)

Missing return type annotation for public function `health_check`

(ANN201)